demisto_sdk/commands/lint/linter.py

```diff
@@ -1582,7 +1580,7 @@ def _get_docker_images_for_lint(
             - If docker_image_flag is 'all', lint will run on:
                 1. Native GA - the native image of the current server version.
                 2. Native Maintenance - the native image of the previous server version.
-                3. Native Dev - The latest tag of the native image for Docker Hub.
```

Comment:
The way to make it run on native:dev is by supplying this image as input (docker_image_flag)?

Reply:
not exactly, that will result in running all (native supported and not) with the docker image provided.
one needs to use
```
demisto-sdk lint -a --docker-image ‘native:target’ --docker-image-target the_specific_image_to_run_on
```